packages/material/test/renderers/MaterialRadioGroupControl.test.tsx

```diff
@@ -0,0 +1,168 @@
+    combineReducers({ jsonforms: jsonformsReducer() }),
+    {
+      jsonforms: {
+        renderers: materialRenderers,
+        fields: materialFields,
+      }
+    }
+  );
+
+  store.dispatch(Actions.init(testData, testSchema, testUiSchema));
+  return store;
+};
+
+describe('Material radio group control tester', () => {
+
+  it('should fail due to enum length', () => {
+    const control: ControlElement = {
+      type: 'Control',
+      scope: '#/properties/options'
+    };
+    expect(
+      materialRadioGroupControlTester(
+        control,
+        {
+          type: 'object',
+          properties: {
+            options: {
+              type: 'string',
+              enum: ['A', 'B', 'C', 'D', 'E', 'F']
+            }
+          }
+        }
+      )
+    ).toBe(-1);
+  });
+});
+
+describe('Material radio group control tester', () => {
+
+  it('should succeed in matching the prop type and enum length', () => {
+    const control: ControlElement = {
+      type: 'Control',
+      scope: '#/properties/options'
+    };
+    expect(
+      materialRadioGroupControlTester(
+        control,
+        {
+          type: 'object',
+          properties: {
+            options: {
+              type: 'string',
+              enum: ['A', 'B', 'C', 'D']
+            }
+          }
+        }
+      )
+    ).toBe(10);
+  });
+});
+
+describe('Material radio group control', () => {
+  it('Radio group should have data option selected', () => {
+    const store = initJsonFormsStore(data, schema, uischema);
+    const tree = TestUtils.renderIntoDocument(
+      <Provider store={store}>
+        <MaterialRadioGroupControl schema={schema} uischema={uischema} data={data} />
+      </Provider>
+    );
+
+    const inputs: HTMLInputElement[] = TestUtils.scryRenderedDOMComponentsWithTag(tree, 'input');
+    const radioButtons = _.filter(
+      inputs,
+      i => i.type === 'radio'
+    );
+    expect(radioButtons.length).toBe(4);
+    // make sure one option is selected and it is "D"
+    const currentlyChecked = _.filter(radioButtons, radio => radio.checked);
+    expect(currentlyChecked.length).toBe(1);
+    expect(currentlyChecked[0].value).toBe('D');
+  });
+});
+
+describe('Material radio group control', () => {
+  it('Radio group should have only one selected option ', () => {
+    const store = initJsonFormsStore(data, schema, uischema);
+    const tree = TestUtils.renderIntoDocument(
+      <Provider store={store}>
+        <MaterialRadioGroupControl schema={schema} uischema={uischema} data={data} />
+      </Provider>
+    );
+
+    const inputs: HTMLInputElement[] = TestUtils.scryRenderedDOMComponentsWithTag(tree, 'input');
+    const radioButtons = _.filter(
+      inputs,
+      i => i.type === 'radio'
+    );
+    expect(radioButtons.length).toBe(4);
+    let currentlyChecked = _.filter(radioButtons, radio => radio.checked);
+    expect(currentlyChecked.length).toBe(1);
+    expect(currentlyChecked[0].value).toBe('D');
```

Comment:
We tested this expectation in the test before this one. Perhaps remove the assertions here and rename the tests accordingly. This test should only whether the binding works as intended.